features/src/main/scala/com/salesforce/op/utils/spark/RichDataset.scala

```diff
@@ -72,7 +71,7 @@ object RichDataset {
     val schemaStr = spark.sparkContext.textFile(schemaPath(path)).collect().mkString
     val schema = DataType.fromJson(schemaStr).asInstanceOf[StructType]
     val origNames = schema.fields.map(_.metadata.getString(OriginalNameMetaKey))
-    val data = spark.read.avro(dataPath(path)).toDF(origNames: _*)
+    val data = spark.read.format("avro").load(dataPath(path)).toDF(origNames: _*)
```

Comment:
nit: Can we leverage https://github.com/apache/spark/blob/master/external/avro/src/main/scala/org/apache/spark/sql/avro/AvroFileFormat.scala#L129 instead of hardcoding "avro" multiple times

Reply:
Seems private. I will see what can be done.